code/game/objects/effects/decals/Cleanable/fuel.dm

```diff
@@ -9,8 +9,10 @@
 /obj/effect/decal/cleanable/liquid_fuel/Initialize(mapload, amt = 1, nologs = 0)
 	. = ..()
 	if(!nologs && !mapload)
-		message_admins("Liquid fuel has spilled in [loc.loc.name] ([loc.x],[loc.y],[loc.z]) (<A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[loc.x];Y=[loc.y];Z=[loc.z]'>JMP</a>)")
-		log_game("Liquid fuel has spilled in [loc.loc.name] ([loc.x],[loc.y],[loc.z])")
+		for (var/mob/living/culprit in range(loc, 3))
```

Comment:
delet this, it's silly.

Use `usr`. And use `log_and_message_admins`.

So replace all of these three lines with:
`log_and_message_admins("spilled liquid fuel", user = usr, location = get_turf(src))`